src/new-api/Anchor.ts

```diff
@@ -76,89 +71,27 @@ export class Anchor {
                     /markers-page-resolution=(\d+)x(\d+)/
                 );
                 console.assert(matches.length === 3);
-                var resolutionW = parseInt(matches[1]);
-                var resolutionH = parseInt(matches[2]);
+                //var resolutionW = parseInt(matches[1]);
+                //var resolutionH = parseInt(matches[2]);
                 var arContext = arSession.arContext;
-                // generate and store the ARjsMultiMarkerFile
-                /*MarkersAreaUtils.storeMarkersAreaFileFromResolution(
```

Comment:
i removed all the markersArea code (markersAreaUtils, markersAreaLearner, ect...). For some reason the code was causing the issue described in the comments above in this PR, for sure my implementation was not correct, but maybe better to re add later...